README.md

```diff
@@ -14,6 +14,16 @@
 
 === An easy to customize, strong type library with built in support for unit-like behavior ===
 
+#### What:
+- Easy support for stron{g|k} typing, with plenty of built-in skills to add functionality your own types.
```

Comment:
- Easy support for stron{g|k} typing, with plenty of built-in skills to add functionality **to** your own types.